rocketpy/rocket/rocket.py

```diff
@@ -1433,7 +1433,16 @@ def add_free_form_fins(
         return fin_set
 
     def add_parachute(
-        self, name, cd_s, trigger, sampling_rate=100, lag=0, noise=(0, 0, 0)
+        self,
+        name,
+        cd_s,
+        trigger,
+        sampling_rate=100,
+        lag=0,
+        noise=(0, 0, 0),
+        parachute_radius=1.5,
+        parachute_height=None,
```

Comment:
I just noticed it now... 

Since it is the Parachute class, there's no need to specify this is the parachute radius or the parachute height.

```suggestion
        radius=1.5,
        height=None,
```

Reply:
I think the simpler naming is better. If the documentation is there we can rely on that